### PR TITLE
Update issue templates

### LINKED ISSUE #16: Make issue template
We need a template to log issues, simplify debugging and make the code base more maintainable. This should at the very least include the following:
- reproduction environment: OS, code that caused the issue/bug
- expected value vs actual value

any other ideas as to what to include in the template?

### PR DESCRIPTION
**Pull request by poftadeh**
Added two templates for bug and feature/enhancement requests. Closes #16.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
```
Comment (manekenpix) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Should we have a "Actual behaviour/result" section as well?

Comment (poftadeh) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Wouldn't that be covered in `Describe the bug`?

Comment (manekenpix) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Yeah, I thought the description would explain the problem, and in a "Actual behaviour/result" section it'd be possible to show (pic or text) the result after running code.

Comment (poftadeh) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Added changes to reflect this. Let me know what you think.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Environment:**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
```
Comment (miggs125) [inline .github/ISSUE_TEMPLATE/bug_report.md]: why is browser needed here?

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (manekenpix) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Nice

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (poftadeh) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Was a part of the default template I edited. Replace with Node version instead?

**Pull request merged**